cloudevents/working-drafts/xml-format.md

```diff
@@ -67,6 +67,10 @@ CloudEvent or CloudEvent batch. Comments MUST be ignored during processing, exce
 for the child element of a `<data>` element containing [XML element data](#33-xml-element-data),
 where all nodes MUST be preserved.
 
+CDATA nodes and regular text nodes MUST be treated interchangably during processing,
```

Comment:
s/interchangably/interchangeably/ 

the word "interchangeably" is throwing me. Do you mean to say that they should be treated the same way in that they're just chunks of text? Or are you trying to say that someone could translate a CDATA node into a text node?

I guess I'm not clear on why we need to call out these two node types at all? What is the XML example that could cause confusion for a receiver?

Reply:
> Do you mean to say that they should be treated the same way in that they're just chunks of text? 

Yes.

> Or are you trying to say that someone could translate a CDATA node into a text node?

When processing the document, they should be treated in the same way, other than within the data element.

> I guess I'm not clear on why we need to call out these two node types at all? What is the XML example that could cause confusion for a receiver?

It means that this is valid, for example:

```xml
<event specversion="1.0">
    <id><![CDATA[abc]]>def</id>
    ... other CE attributes as normal ...
</event>
```

... and should be processed exactly as if it were written like this:

```xml
<event specversion="1.0">
    <id>abcdef</id>
    ... other CE attributes as normal ...
</event>
```

or

```xml
<event specversion="1.0">
    <id><![CDATA[abcdef]]></id>
    ... other CE attributes as normal ...
</event>
```


I want us to have conformance tests with examples like that, but I don't *think* that sort of example is worth including in the format spec itself.

Reply:
Thanks for the clarification and that makes sense. Why did this come up as an issue rather than it just being obvious? Or, what other choice might someone make?